### DISCUSSION TITLE
Use CascadeSelect with Angular Signal

### DISCUSSION THREAD

**Discussion by danilokorber** [primefaces/community#846] [PrimeNG]
I am trying to use CascadeSelect and point the options to an Angular Signal:

```
readonly store = inject(Store);
readonly countries = this.store.selectSignal(
    (state) => state.countries.items ?? []
);
```

In my template:

```
<p-cascadeSelect
    [options]="countries()"
    optionLabel="name"
    optionGroupLabel="country"
    [optionGroupChildren]="['cities']"
  ></p-cascadeSelect>
  
  {{ countries() | json }}
```
  
  On the bottom of the screen I can see all loaded contries with its cities, but the CascadeSelect component is never populated. Are Signals already supported by PrimeNg?

**Comment by cetincakiroglu** [comment]
Hi,

You can use signals, please see the example https://stackblitz.com/edit/ztqddv?file=src%2Fapp%2Fdemo%2Fcascade-select-basic-demo.ts,src%2Fapp%2Fdemo%2Fcascade-select-basic-demo.html

Could you please provide a stackblitz reproducer of your use case so we can identify the issue?

**Reply by danilokorber** [reply]
I was not able to reproduce my problem in StackBlitz. So I guess I have to debug my code...
